kubernetes-manifests/loadgenerator.yaml

```diff
@@ -72,7 +69,6 @@ apiVersion: v1
 kind: Service
 metadata:
   name: loadgenerator-external
-  namespace: demo-env
```

Comment:
on docker for mac, this will not work, because 2 TCP services cannot have the same port of 80 (at least on the version that I seem to have) - frontend-external also is a loadbalancer with port 80

Reply:
fixed. had to introduce another bash hack. will come need to come up with better approach later